readme.md

```diff
@@ -169,6 +169,8 @@ It exports the additional types [`Options`][api-options] and
 To type `file.data.matter`, you can augment `DataMap` from `vfile` as follows:
 
 ```ts
+export {}
```

Comment:
I think that if we have it like this, people will just copy it over without thinking.
In `unified`, I just added this in examples. What do you think about having it?

```ts
export {} // You may not need this, but it makes sure the file is a module.
```

Preferably at the bottom instead of the top for me, because it’s optional, and has more to do with that TS doesn’t compile this example as-is, but will compile it often.

Thanks!

Reply:
@iojcde Can you add the comment?